library/pod-security-policy/seccomp/template.yaml

```diff
@@ -67,215 +67,349 @@ spec:
+          - name: initContainers
+            expression: 'has(variables.anyObject.spec.initContainers) ? variables.anyObject.spec.initContainers : []'
+          - name: ephemeralContainers
+            expression: 'has(variables.anyObject.spec.ephemeralContainers) ? variables.anyObject.spec.ephemeralContainers : []'
+          - name: allowAllProfiles
+            expression: |
+              has(variables.params.allowedProfiles) && variables.params.allowedProfiles.exists(profile, profile == "*")
+          - name: exemptImagePrefixes
+            expression: |
+              !has(variables.params.exemptImages) ? [] :
+                variables.params.exemptImages.filter(image, image.endsWith("*")).map(image, string(image).replace("*", ""))
+          - name: exemptImageExplicit
+            expression: |
+              !has(variables.params.exemptImages) ? [] : 
+                variables.params.exemptImages.filter(image, !image.endsWith("*"))
+          - name: exemptImages
+            expression: |
+              (variables.containers + variables.initContainers + variables.ephemeralContainers).filter(container,
+                container.image in variables.exemptImageExplicit ||
+                variables.exemptImagePrefixes.exists(exemption, string(container.image).startsWith(exemption)))
+          - name: unverifiedContainers
+            expression: |
+              (variables.containers + variables.initContainers + variables.ephemeralContainers).filter(container,
+                !variables.allowAllProfiles &&
+                !(container.image in variables.exemptImages))
+          - name: inputAllowedProfiles
+            expression: |
+              !has(variables.params.allowedProfiles) ? [] : variables.params.allowedProfiles
+          - name: allowedLocalhostFiles
+            expression: |
+              has(variables.params.allowedLocalhostFiles) ? variables.params.allowedLocalhostFiles : []
+          - name: allowedProfilesTranslation
+            expression: |
+              (variables.inputAllowedProfiles.filter(profile,
+              profile != "Localhost").map(profile, profile == "Unconfined" ? "unconfined" : profile)) + 
+              (variables.inputAllowedProfiles.exists(profile, profile == "RuntimeDefault") ? ["runtime/default", "docker/default"] : [])
+          - name: allowSecurityContextLocalhost
+            expression: |
+              variables.inputAllowedProfiles.exists(profile, profile == "Localhost")
+          - name: derivedAllowedLocalhostFiles
+            expression: |
+              variables.allowSecurityContextLocalhost ? variables.params.allowedLocalhostFiles.map(file, "localhost/" + file) : []
+          - name: localhostWildcardAllowed
+            expression: |
+              variables.inputAllowedProfiles.exists(profile, profile == "localhost/*") || variables.derivedAllowedLocalhostFiles.exists(profile, profile == "localhost/*")
+          - name: allowedProfiles
```

Comment:
What about users who input `localhost/<some profile>`? It looks like that's getting dropped?

Reply:
I don't think so. I added the same in constraint - https://github.com/open-policy-agent/gatekeeper-library/pull/540/files#diff-58803b65572868312ed7364039674b1871f873f15cfd13aa29c8b9adb42e2901 - and added the examples to test `Localhost` profile. PTAL.

Reply:
Oops, missed it being added in a lower line

Reply:
No worries! Good catch! I updated the code.